test/integration/emulator/SfuClient.h

```diff
@@ -576,6 +606,12 @@ class SfuClient : public transport::DataReceiver
                     }
                 }
             }
+
+            if (rtpHeader->padding == 1)
```

Comment:
Should we check that it is RTX payload type also. Padding is strictly allowed and ignored in real vp8 packets.

Reply:
Addressed.